### DISCUSSION TITLE
Release Hotfix Including Commit to Address Critical Print Range Error

### DISCUSSION THREAD

**Discussion by yuki2006** [qax-os/excelize#1703] [General]
thx very useful libs.

In cases like setting the print range, it can be fatal due to errors, so let's do a hotfix release

https://github.com/qax-os/excelize/commit/a0a7d5cdbb092d421bdb0b7bce82a93fe350af5b

**Comment by xuri** [comment]
Thanks for your feedback. This patch will be released in the next version. Please upgrade the master branch code and lock the package version by Go Modules before v2.8.1 released.
